frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -210,6 +210,23 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  // Download External Cluster Metadata
+  {
+    type: 'ClusterServiceVersion/Action',
+    properties: {
+      kind: 'StorageCluster',
+      label: 'Download Cluster Metadata',
+      apiGroup: models.OCSServiceModel.apiGroup,
+      callback: (_kind, _obj) => () =>
```

Comment:
disable eslint for this line

Reply:
Do we really need to declare those `_kind` and `_obj` parameters here?

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts#L13